contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -533,6 +533,8 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
         _validateExchangeSettings(_exchangeSettings);
         
         exchangeSettings[_exchangeName] = _exchangeSettings;
+        exchangeSettings[_exchangeName].exchangeLastTradeTimestamp = 0;
```

Comment:
Can we save gas by setting each variable individually instead of setting all of them and then over-writing? Not 100% sure this saves gas but assume it does